### PR TITLE
Refactored navigation to get its items from properties

### PR DESCRIPTION
**Pull request by larsblumberg**

### CODE REVIEW

File: public/app/screens/main/main-screen.jsx [hunk 1]
```diff
@@ -7,12 +7,21 @@ var Navigation = require('common/regions/navigation');
 var style = require("./style.scss");
 
 var MainScreen = React.createClass({
+    
+    getInitialState: function() {
+        return {
+            navItems: [
```
Comment (dev-tim) [inline public/app/screens/main/main-screen.jsx]: Is it a good place to store config for navigation?

Comment (larsblumberg) [inline public/app/screens/main/main-screen.jsx]: What do you suggest?

Comment (dev-tim) [inline public/app/screens/main/main-screen.jsx]: It need's to be in sync with routes, I would suggest to have such config in routes file.

Comment (mr-mig) [inline public/app/screens/main/main-screen.jsx]: Create a store.
If you wish you can merge it, but create an issue with the code reference.

**Pull request merged**